Formula/snobol4.rb

```diff
@@ -19,8 +18,58 @@ class Snobol4 < Formula
     sha256 yosemite:       "6903e1b05a795eae13f2f97fd2f1f4b883b03e1c94ba28e3747b3df98c6a955d"
   end
 
+  depends_on "libffi"
+  depends_on "openssl@1.1"
+  depends_on "readline"
+  depends_on "sqlite"
+  depends_on "zlib"
```

Comment:
Do these (ignoring OpenSSL) need the latest version or can the macOS version work? Apple's libffi for one is probably better than ours when it comes to arm64 trampoline handling.

Reply:
If the answer is "not sure", that's fine - I can test it out locally.